subgraph/subgraph.yaml

```diff
@@ -5,11 +5,11 @@ schema:
 dataSources:
   - kind: ethereum
     name: Unlock
-    network: goerli
+    network: mainnet
```

Comment:
this is kind of annoying... we need to check changes to the subgraph.yml file every time